rero_invenio_base/cli/es/snapshot/__init__.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+#
+# RERO Invenio Base
+# Copyright (C) 2022 RERO.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Click elasticsearch snapshot command-line utilities."""
+
+import json
+from datetime import datetime
+
+import click
+from elasticsearch_dsl import Index
+from flask.cli import with_appcontext
+from invenio_search import current_search, current_search_client
+
+from ...shared import abort_if_false
+from .repository import repository
+
+
+@click.group()
+def snapshot():
+    """Elasticsearch snapshot commands."""
+
+
+snapshot.add_command(repository)
+
+
+@snapshot.command('list')
+@with_appcontext
+@click.option('-n', '--name', help="default=_all", default='_all')
+@click.argument('repository')
+def list_snapshot(repository, name):
```

Comment:
cosmetic : Personnaly, I don't like to put logical code into "__init__.py" when it's not necessary. Could be better to put these methods into a `cli.py` fil, import methods and use `__all__` buildin variable ?